src/models/slices/filtersearch.ts

```diff
@@ -0,0 +1,10 @@
+import { FilterSearchResponse } from '@yext/answers-core';
+
+interface FilterSearchState {
+  query: string,
+  results?: FilterSearchResponse
+}
+
+export interface FilterSearchStates {
+  [filterSearchId: string]: FilterSearchState
```

Comment:
I'm thinking we might want to call this filterSearchCollectionId so that we're consistent with filterCollectionId

Reply:
I was thinking the value here is not really a collection since it's a state of query + result, unlike the mapping in staticFilter where it's a collection/list of selectable filters, so maybe that naming wouldn't fit so well.

Reply:
in that case I think  `filterSearchId` works